jit/v2/JIT_v2.0.0.yaml

```diff
@@ -2862,82 +2862,82 @@ components:
+        loadUnits:
+          $ref: '#/components/schemas/LoadUnits'
+        dischargeUnits:
+          $ref: '#/components/schemas/DischargeUnits'
 
-    LoadMoves:
+    LoadUnits:
       type: object
-      title: Load Moves
+      title: Load Units
       description: |
         Forecasted total units of containers to move for loading, empty or laden, at a terminal, covering various types such as Dry, Reefer, Out-of-Gauge (OOG)
       properties:
-        ladenContainerMoves:
-          $ref: '#/components/schemas/LadenContainerMoves'
-        emptyContainerMoves:
-          $ref: '#/components/schemas/EmptyContainerMoves'
-        reeferPluggedContainerMoves:
-          $ref: '#/components/schemas/ReeferPluggedContainerMoves'
+        ladenContainerUnits:
+          $ref: '#/components/schemas/LadenContainerUnits'
+        emptyContainerUnits:
+          $ref: '#/components/schemas/EmptyContainerUnits'
+        reeferPluggedContainerUnits:
+          $ref: '#/components/schemas/ReeferPluggedContainerUnits'
         oogUnits:
           type: integer
           format: int32
           description: |
             Forecasted Out-of-gauge (OOG) units
           example: 2
-    DischargeMoves:
+    DischargeUnits:
       type: object
-      title: Discharge Moves
+      title: Discharge Units
       description: |
         Forecasted total units of containers to move for discharging, empty or laden, at a terminal, covering various types such as Dry, Reefer, Out-of-Gauge (OOG)
       properties:
-        ladenContainerMoves:
-          $ref: '#/components/schemas/LadenContainerMoves'
-        emptyContainerMoves:
-          $ref: '#/components/schemas/EmptyContainerMoves'
-        reeferPluggedContainerMoves:
-          $ref: '#/components/schemas/ReeferPluggedContainerMoves'
+        ladenContainerUnits:
+          $ref: '#/components/schemas/LadenContainerUnits'
+        emptyContainerUnits:
+          $ref: '#/components/schemas/EmptyContainerUnits'
+        reeferPluggedContainerUnits:
+          $ref: '#/components/schemas/ReeferPluggedContainerUnits'
         oogUnits:
           type: integer
           format: int32
           description: |
             Forecasted Out-of-gauge (OOG) units
           example: 2
 
-    LadenContainerMoves:
+    LadenContainerUnits:
       type: object
-      title: Laden Container Moves
+      title: Laden Container Units
       description: |
         Forecasted laden containers
       oneOf:
-        - $ref: '#/components/schemas/TotalMoves'
-        - $ref: '#/components/schemas/ContainerMoves'
+        - $ref: '#/components/schemas/TotalUnits'
+        - $ref: '#/components/schemas/ContainerUnits'
 
-    ContainerMoves:
+    ContainerUnits:
       type: object
-      title: Container Moves
+      title: Container Units
       description: |
         Forecasted units of containers to move for `20-foot`, `40-foot` and `45-foot` containers
       properties:
```

Comment:
**In line 2948** and similar ones where we have "Forecasted moves of `20-foot` containers", or 40 or 45...

Could you change to "Forecasted units of `20-foot` containers" to align with units?. Don't think is needed to add "to move" at the end of these descriptions , they are already within a Container units object defined as "units of containers to move", would be redundant.